frontend/src/pages/coding/[id].tsx

```diff
@@ -199,16 +247,26 @@ const CodingPage = () => {
 						/>
 						<h1 className="font-bold text-xl">{project?.title}</h1>
 					</div>
-					<div className="flex items-center self-end md:self-center">
-						<Button
-							size={"sm"}
-							className="gap-2 bg-blue-500 hover:bg-blue-500/80"
-							onClick={saveCode}
-						>
-							<Save />
-							Enregistrer
-						</Button>
-					</div>
+					{project && userId === project.user.id && (
+						<div className="flex items-center self-end md:self-center gap-5">
+							<div className="flex items-center space-x-2">
```

Comment:
```suggestion
						<div className="flex items-center self-end md:self-center gap-5">
								<LikeButton project={project} userId={userId} />
								<div className="flex items-center space-x-2">
```
C'est dommage de mettre le bouton de like en bas, il est invisible, ne serait-il pas mieux vers là?
![image](https://github.com/user-attachments/assets/319cb487-40d9-4b73-bfdd-9a3d3a3594bc)

Reply:
Yes mais vu qu'il y aura les commentaires en bas à voir ce que ça donne par la suite, je me suis inspiré d'i*Instagram (avec j'aime, commentaires et partage), on demandera aux autres ce qu'ils en pensent

Reply:
ha oui pas pensé au commentaires, a voir plus tard alors ;)